app-arch/lbzip2/lbzip2-2.5_p20181227-r3.ebuild

```diff
@@ -29,6 +29,12 @@ src_prepare() {
 src_configure() {
 	use static && append-ldflags -static
 
+	# fix clang miscompilation: #910438
+	# see also: https://github.com/llvm/llvm-project/issues/87189
+	if use amd64 || use x86; then
+		tc-is-clang && append-flags "-mno-avx512f"
```

Comment:
Let's just do with no amd64/x86 check:
`tc-is-clang && append-flags $(test-flag-CC -mno-avx512f)`

Reply:
Done (slightly different as `test-flag-CC` returns shell true)